packages/fuselage-ui-kit/src/surfaces/createSurfaceRenderer.tsx

```diff
@@ -4,6 +4,7 @@ import type { ComponentType, ReactElement } from 'react';
 export const createSurfaceRenderer = <
   S extends UiKit.SurfaceRenderer<ReactElement>
 >(
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
what changed that made this required? has some naming convention change?

Reply:
Yep, for some reason it says it should be on camel case, but it's on Pascal 🤔